src/js/wrapper.ts

```diff
@@ -381,7 +381,6 @@ export const NATIVE: SentryNativeWrapper = {
       ...breadcrumb,
       // Process and convert deprecated levels
       level: breadcrumb.level ? this._processLevel(breadcrumb.level) : undefined,
-      data: breadcrumb.data ? normalize(breadcrumb.data) : undefined,
```

Comment:
Did we move this somewhere?

Reply:
Yes, it's now here.
https://github.com/getsentry/sentry-react-native/blob/699a4f21602f14c6edc6ab9ae42d0e430e9e27c5/src/js/scope.ts#L66